production/helm/loki/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: "v1"
 name: loki
-version: 0.22.0
```

Comment:
I think we wait until a release before we update the Helm charts? Not 100% sure. /cc @slim-bean

Reply:
nope, we update the helm chart every time there is a change to it independently of the release

Reply:
I feel like I'm missing something super obvious here, I'm not sure what changed that makes the charts need a version bump

Reply:
ha, no you're not, I assumed there was a config change in the charts somewhere

yeah there is no reason to bump the charts if we didn't change them

That being said, because it's helm, do we need to expose this new config item somewhere to make it easy for people to change?

Reply:
After our discussion, we're going to bump those later